server/views/common/reportAProblem.njk

```diff
@@ -1,6 +1,41 @@
 {% extends "../partials/layout.njk" %}
 
 {% block pageContent %}
-  <h1 class="govuk-heading-l">Report a problem</h1>
-  {{ content | safe }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Report a problem</h1>
+
+      <h2 class="govuk-heading-m">NPS staff</h2>
+
+      <p class="govuk-body">To report a problem with this digital service, please contact the helpdesk on <a href="tel:+44800 917 5148">0800 917 5148</a></p>
+
+      <h2 class="govuk-heading-m">Suppliers</h2>
+
+      <p class="govuk-body">To report a problem with this digital service, please contact your helpdesk who will contact the digital service team if necessary. </p>
+      <p class="govuk-body">If your organisation does not have an IT helpdesk, please contact the helpdesk on <a href="tel:+44800 917 5148">0800 917 5148</a></p>
+
+      <h2 class="govuk-heading-m">Learning and resources for NPS staff</h2>
+
+      <ul class="govuk-list govuk-list--bullet">
+        <li><a class="govuk-link" href="https://intranet.noms.gsi.gov.uk/corporate/probation-changes/Commissioned-Rehabilitative-Services/your-learning-probation-practitioners-and-contract-managers">NPS Intranet Learning Section</a></li>
```

Comment:
Do we want these links to open in new tabs (ie. should we add `target="_blank" rel="noopener noreferrer"` to them?)

Reply:
last time i had that discussion it was discouraged.